packages/chart/src/components/Legend.jsx

```diff
@@ -170,13 +174,25 @@ const Legend = () => {
         </LegendOrdinal>
       </aside>
     ) : (
-      <aside id='legend' className={containerClasses.join(' ')} role='region' aria-label='legend' tabIndex={0}>
+      <aside
+        id='legend'
+        className={containerClasses.join(' ')}
+        role='region'
+        aria-label='legend'
+        tabIndex={0} // eslint-disable-line
+      >
         {config.boxplot.legend.displayHowToReadText && <h3>{config.boxplot.legend.howToReadText}</h3>}
       </aside>
     )
   return (
     config.visualizationType !== 'Box Plot' && (
-      <aside id='legend' className={containerClasses.join(' ')} role='region' aria-label='legend' tabIndex={0}>
+      <aside
+        id='legend'
+        className={containerClasses.join(' ')}
+        role='region'
+        aria-label='legend'
+        tabIndex={0} // eslint-disable-line
```

Comment:
fix: remove this mute since we need to address the accessibility issue.

Reply:
removed